packages/site/src/components/FeedbackBanner.tsx

```diff
@@ -37,7 +37,7 @@ export const FeedbackBanner = () => {
         padding: "var(--space-small)",
         transition: "transform 0.3s ease-in-out",
```

Comment:
Could use our timing tokens here

```suggestion
        transition: "transform var(--timing-base) ease-in-out",
```